.github/workflows/finishReleaseCycle.yml

```diff
@@ -88,6 +88,7 @@ jobs:
     steps:
       - run: |
           gh issue list \
+            --repo 'Expensify/App' \
             --state all \
             --label DeployBlockerCash \
             --json number \
```

Comment:
NAB 
xargs option:
```
xargs -n1 -P4 -I{} gh issue edit {} --repo 'Expensify/App' --remove-label DeployBlockerCash
```

Reply:
lol, you are probably right about this, and it should be a blocker 😬